msg_modification/tests_encode_to_chunked.py

```diff
@@ -29,7 +29,17 @@ class MessageTransformations(tester.TempestaTest):
     tempesta = {
         'config' :
         """
-        server ${server_ip}:8000;
+        listen 80;
+        
+        srv_group * {
+            server ${server_ip}:8000;
+        }
+        vhost * {
+            proxy_pass *;
+        }
+        http_chain {
+            -> *;
+        }
```

Comment:
IIRC there were changes that we now require explicit statement for `http_chain`, but it should be possible to handle default vhost and default server group, so I'd prefer to leave several tests for the cases. See https://github.com/tempesta-tech/tempesta/wiki/Backend-servers and https://github.com/tempesta-tech/tempesta/wiki/Virtual-hosts-and-locations . If there aren't options for default server group and vhost, then we need to fix our wiki

Reply:
`        http_chain {
            -> *;
        }`
        
Removed

Reply:
That was basically about the difference of having "*" and "default" in tests :)

Reply:
I created issue https://github.com/tempesta-tech/tempesta/issues/1650 , it seems we do have some bug here